lib/js/components/Buttons/HelpButton/HelpButton.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+	<tooltip :text="tooltipText" :inline="true">
+		<icon-button
+			:icon="ICONS.FA_CIRCLE_QUESTION"
+			:color="ICON_BUTTON_COLORS.NEUTRAL_WEAK"
+			:size="ICON_BUTTON_SIZES.X_SMALL"
+			:touchable="false"
+			v-bind="$attrs"
+			@click="isOpen = true"
+		/>
+	</tooltip>
+	<slot v-if="isOpen" name="modal" @close="onClose"></slot>
```

Comment:
https://bethinkteam.slack.com/archives/CFBLF5RGA/p1749736300369779

Reply:
added modalTitle prop and modalContent slot